sdcm/cluster.py

```diff
@@ -3284,6 +3284,18 @@ def check_nodes_up_and_normal(self, nodes=None, verification_node=None):
         if not all(up_statuses):
             raise ClusterNodesNotReady("Not all nodes joined the cluster")
 
+    def get_nodes_up_and_normal(self, verification_node=None):
```

Comment:
maybe have also `get_nodes_down()` so you can get the host_id ?

Reply:
I added a different helper function